src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: transform ease 3s;
```

Comment:
The transition duration of 3 seconds is too long, which can lead to a poor user experience. Consider reducing the duration to something around 0.3 seconds.